README.md

```diff
@@ -124,7 +124,7 @@ For information about contributing to OpenTelemetry Python, see [CONTRIBUTING.md
 
 We meet weekly on Thursdays at 9AM PST. The meeting is subject to change depending on contributors' availability. Check the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com) for specific dates.
 
-Meetings take place via [Zoom video conference](https://zoom.us/j/6729396170). The passcode is _77777_.
+Meetings take place via [Zoom video conference](https://zoom.us/j/8287234601?pwd=YjN2MURycXc4cEZlYTRtYjJaM0grZz09). The passcode is _77777_.
```

Comment:
Do we want to include the encoded password in the url, especially when we already provide the password?

Reply:
I thought we might as well since the encoded password is already on [the public calendar](https://calendar.google.com/calendar/u/0/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60@group.calendar.google.com&pli=1).

If someone looks at the ID in the link and inputs that the password is still useful! So I would say yes since opening it without the need for a password is just happily convenient.